en/containers-run-as-non-root-user.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Run TiDB Operator and TiDB Clusters as a Non-root User
+summary: Make TiDB Operator related containers run as a non-root user
+aliases: ['/docs/tidb-in-kubernetes/dev/containers-run-as-non-root-user/']
```

Comment:
This is a new doc and it does not replace any old doc, so the alias is not needed.
```suggestion
```